src/Autofac.Extras.MvvmCross/AutofacMvxIocProvider.cs

```diff
@@ -521,5 +587,59 @@ public bool TryResolve(Type type, out object resolved)
 
             return _container.TryResolve(type, out resolved);
         }
+
+        private void EnablePropertyInjection(Type type, IRegistrationBuilder<object, IConcreteActivatorData, SingleRegistrationStyle> registrationBuilder)
+        {
+            var injectableProperties = type
+                .GetProperties(BindingFlags.Public | BindingFlags.Instance | BindingFlags.FlattenHierarchy)
+                .Where(p => p.PropertyType.GetTypeInfo().IsInterface)
+                .Where(p => p.IsConventional())
+                .Where(p => p.CanWrite);
+
+            switch (PropertyInjectionOptions.InjectIntoProperties)
+            {
+                case MvxPropertyInjection.MvxInjectInterfaceProperties:
+                    if(_customInjectorAttributeType != null)
+                        injectableProperties = injectableProperties
+                            .Where(p => p.GetCustomAttributes(typeof(MvxInjectAttribute), false).Any() ||
+                                        p.GetCustomAttributes(_customInjectorAttributeType, false).Any());
```

Comment:
The reason for the custom attribute is simple.
We use some of our services on the server as well and I do NOT want to include MvvmCross in an ASP.NET MVC Application ;-)